frontend/src/pages/Login.jsx

```diff
@@ -22,15 +22,21 @@ import { useNavigate } from 'react-router-dom';
 
 import { IMG_SPLASH, LOGIN_LOGO, URL_BASE_INV } from '../globals/urls';
 import { useLogin } from '../hooks/mutations';
+import { isAuthenticated } from '../hooks/auth/authHook';
 
 export const Login = () => {
+  const navigate = useNavigate();
+  useEffect(() => {
+    if (isAuthenticated() === 'true') {
+      navigate(URL_BASE_INV);
+    }
+  }, [navigate]);
```

Comment:
can we put this useEffect after all the const declerations

Reply:
Have shifted it down and removed unused packages.